package.json

```diff
@@ -1,35 +1,37 @@
 {
-  "name": "html5-history-api",
-  "npmName": "html5-history-api",
-  "npmFileMap": [{
-    "basePath": "/",
-    "files": [
-      "*.js"
-    ]
-  }],
-  "version": "4.2.5",
-  "description": "HTML5 History API expansion for browsers not supporting pushState, replaceState",
-  "main": "history.js",
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/devote/HTML5-History-API.git"
-  },
-  "keywords": [
-    "javascript",
-    "html5 history api",
-    "hashchange",
-    "popstate",
-    "pushstate",
-    "replacestate",
-    "hashes",
-    "hashbang",
-    "history",
-    "html5",
-    "devote"
-  ],
-  "author": "Dmitrii Pakhtinov",
-  "licenses": [
-    "GPL-3.0",
-    "MIT"
-  ]
+	"name": "html5-history-api",
+	"version": "4.2.5",
+	"description": "HTML5 History API expansion for browsers not supporting pushState, replaceState",
+	"keywords": [
+		"javascript",
+		"html5 history api",
+		"hashchange",
+		"popstate",
+		"pushstate",
+		"replacestate",
+		"hashes",
+		"hashbang",
+		"history",
+		"html5",
+		"devote"
+	],
+	"licenses": [
+		{},
```

Comment:
Hmm, I haven't noticed it. I'm using some new add-on for Atom and it rewrites packages.json files as soon as I open it. I guess it deleted the line. I'll restore it in a moment.